package.json

```diff
@@ -144,6 +149,11 @@
 					"when": "viewItem == libertyMavenProject || viewItem == libertyGradleProject || viewItem == libertyMavenProjectContainer || viewItem == libertyGradleProjectContainer",
 					"group": "libertyCore@1"
 				},
+				{
+					"command": "liberty.dev.debug",
+					"when": "viewItem == libertyMavenProject || viewItem == libertyGradleProject || viewItem == libertyMavenProjectContainer || viewItem == libertyGradleProjectContainer",
+					"group": "libertyCore@1"
+				},
```

Comment:
Do we have control over which order the commands appear in the dashboard? Currently looks to be alphabetical, but ideally "Attach debugger" would fall below the "Start" commands

![image](https://user-images.githubusercontent.com/26146482/199541244-0481e3bd-67c9-4e01-b8a1-13c074c7cf54.png)

Reply:
Addressed